app/views/plans/_activities_card.html.haml

```diff
@@ -1,7 +1,7 @@
 #plan-activities-card.sa-activities-grid.border.my-4.p-4
   %h6= Plan.human_attribute_name('activities')
 
-  #plan-activities-list
+  #plan-activities-list{ data: { activity_active_tag_associations_count: Activity.new.active_tag_associations.count } }
```

Comment:
@SimonSharville here it is :) remember there is +1 becasue we have time worked cell